vm-virtio/src/net.rs

```diff
@@ -659,6 +678,18 @@ impl VirtioDevice for Net {
             }
             self.queue_evts = Some(tmp_queue_evts);
 
+            let ctrl_q = if (self.acked_features & 1 << VIRTIO_NET_F_CTRL_VQ) != 0 {
+                if queues.len() % 2 != 0 {
+                    let cvq_queue = queues.pop().unwrap();
```

Comment:
Done! While for Tx/Rx queues, it will gain better code readability, but for control queue, I do not see much difference, what's your concern on pop? for it is possible to return None?